spec/istring.dd

```diff
@@ -0,0 +1,228 @@
+Ddoc
+
+$(SPEC_S Interpolation Expression Sequence,
+
+$(HEADERNAV_TOC)
+
+$(P Interpolation Expression Sequences (IES) are expressions that intersperse
+string literal data, and values. An interpolation expression is written like a
```

Comment:
i'd delete the comma lol

otherwise tl;dr but no gross errors jumped out at me